vmdb/app/models/rss_feed.rb

```diff
@@ -82,7 +82,7 @@ def self.roles
     # Tag.tags(:ns=>"managed", :cat=>"roles").each {|t|
     #   roles.push(t.name.split("/").last)
     # }
-    Tag.find(:all, :conditions=>"name like '/managed/roles/%'").each {|t| roles.push(t.name.split("/").last)}
+    Tag.where("name like '/managed/roles/%'").each { |t| roles.push(t.name.split("/").last) }
```

Comment:
Is all this just a collect on a pluck?  `Tag.where("name...").pluck(:name).collect { |n| n.split("/").last }`